org.gnome.Geary.json

```diff
@@ -0,0 +1,113 @@
+/* flatpak-builder config for Geary. */
+{
+    "app-id": "org.gnome.Geary",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "geary",
+    "tags": [],
+    "rename-icon": "geary",
+    "copy-icon": true,
+    "finish-args": [
+        /* X11 + XShm access */
+        "--share=ipc", "--socket=x11",
+
+        /* Wayland access */
+        "--socket=wayland",
+
+        /* OpenGL access for WK2 */
+        "--device=dri",
+
+        /* Bus access */
+        "--socket=session-bus",
+
+        /* Pulseaudio */
+        "--socket=pulseaudio",
+
+        /* Needs to talk to the network: */
+        "--share=network",
+
+        /* Notification access */
+        "--talk-name=org.freedesktop.Notifications",
+
+        /* Secrets access */
+        "--talk-name=org.freedesktop.secrets",
+
+        /* Needed for dconf to work */
+        "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+
+        /* Let view source keep on working as-sis for now. Bug 779311. */
+        "--filesystem=/tmp"
```

Comment:
Usually the solution to something wanting `/tmp` is to set `TMPDIR=$XDG_CACHE_HOME`: https://github.com/flathub/com.discordapp.Discord/blob/master/com.discordapp.Discord.json#L52